File: src/main/java/pe/nanamochi/poc_osu_server/packets/server/ChannelInfoCompletePacket.java
package pe.nanamochi.poc_osu_server.packets.server;

import lombok.Data;
import lombok.NoArgsConstructor;
import pe.nanamochi.poc_osu_server.packets.Packet;
import pe.nanamochi.poc_osu_server.packets.Packets;

@Data
@NoArgsConstructor
public class ChannelInfoCompletePacket implements Packet {
    @Override
    public Packets getPacketType() {
        return Packets.BANCHO_CHANNEL_INFO_COMPLETE;
    }
}

File: banchus-packet/src/main/kotlin/pe/nanamochi/banchus/packets/server/ChannelInfoCompletePacket.kt
package pe.nanamochi.banchus.packets.server

import java.io.OutputStream
import pe.nanamochi.banchus.core.PacketType
import pe.nanamochi.banchus.core.ServerPacket
import pe.nanamochi.banchus.io.DataWriter

class ChannelInfoCompletePacket : ServerPacket() {
    override val type = PacketType.BANCHO_CHANNEL_INFO_COMPLETE

    override fun write(writer: DataWriter, out: OutputStream) {}
}
